src/api/queries/blockchainQueries.js

```diff
@@ -113,17 +99,18 @@ ApiClient_.prototype.fetchTokenAgeConsumed = function (slug, from, to) {
 ApiClient_.prototype.fetchMvrvRatio = function (slug, from, to) {
   var query = {
     'query': '{\
-       mvrvRatio(slug: "' + slug + '",\
-                 from: "' + toUTC_(from) + '",\
-                 to: "' + toUTC_(to) + '",\
-                 interval: "1d") {\
-         ratio\
-         datetime\
-       }\
-     }'
+       getMetric(metric: "mvrv_usd_1d") {\
```

Comment:
the default should be `mvrv_usd`

Reply:
This code is not needed, check the above comment.